### PR TITLE
Major changes to cloth rag chokehold behavior

### PR DESCRIPTION
**Pull request by colossusqw**
[LABEL][balance][chemistry]
## About the PR 
This PR alters the behavior of cloth rags and handkerchiefs in 3 distinct ways:
- Increases the maximum held volume from 10 to 20, and the grab chemical output from 0.5 to 1.5(ripped cloth pieces) or 2(bartender handkerchiefs).
- Removes the ingest effects of chemicals inhaled through the grab.
- Makes it so getting someone on an aggressive grab causes you to transfer chemicals after a 2 second delay.

## Why's this needed? 
I don't remember ever seeing anyone use the cloth rags besides very few and far between moments, since their chemical transfer requires a chokehold, an action that both kills your target, and is pretty much a win condition anyways.

In pretty much all situations, rags are just not worth it even when compared to just hand grabbing your opponent, besides the honestly kinda janky stacking of ingest effects(looking at you, phlog and bath salts hellspam). This PR also removes the aforementioned ingest effects from the rags, to incentivise their use as a way to put chems in people, and not just deal 10 damage per cycle to them.

As it stands, these changes actually allow you to nonlethaly tackle targets in a way that requires more commitment and is more interesting to go against than something instant and unavoidable like scalpel dipping, but cuts down on some of the more jank and overly damaging effects like phlog ingest spam.

## Changelog 

```changelog
(u)Colossusqw
(+)Changed cloth rag grabs to begin transfering chems on aggressive grabs, after a two second delay.
(+)Cloth rags now can hold up to 20 units of chems, and transfer 1.5(ripped cloth rags) or 2(bartender handkerchiefs) per cycle, but don't cause ingest effects anymore.
```

### CODE REVIEW

File: code/obj/item/cloths.dm [hunk 1]
```diff
@@ -43,7 +43,7 @@ ABSTRACT_TYPE(/obj/item/cloth)
 
 /obj/item/cloth/New()
 	..()
-	src.create_reagents(10)
+	src.create_reagents(20)
```
[the remark this answers is not in the document]
Comment (colossusqw) [reply code/obj/item/cloths.dm:46]: That seems pretty interesting, although the 10 unit limit is currently a part of what limit the rag's usage to some of the more 'gamer' high lethality chems, i wanted to expand their usefulness into less all around lethal chems like capu/ether. If you know a way around that i'd like to know as well!

[the remark this answers is not in the document]
Comment (colossusqw) [reply code/obj/item/cloths.dm:46]: Trying something with transfer speeds.

**Comment by TobleroneSwordfish** [comment]
I kind of don't like the action bar, how awful would it be if it just applied immediately on aggressive grab?

**Comment by colossusqw** [comment]
> I kind of don't like the action bar, how awful would it be if it just applied immediately on aggressive grab?

I could try that, it doesn't seem like it'd break much stuff balance wise considering the ingest effects are gone..

**Pull request merged**